service/src/main/java/bio/terra/workspace/service/resource/controlled/model/ControlledResource.java

```diff
@@ -51,7 +65,7 @@ public ControlledResource(DbResource dbResource) {
    *
    * @param fields container for building WsmResource and Controlled resource
    */
-  public ControlledResource(ControlledResourceFields fields) {
```

Comment:
why is this visibility changed to protected?

Reply:
I expect this constructor only to be used from subclasses.
Do you think it needs to be public?